src/net_processing.cpp

```diff
@@ -1998,7 +1998,7 @@ void static ProcessOrphanTx(CConnman* connman, CTxMemPool& mempool, std::set<uin
  * @param[out]  filter_index    The filter index, if the request can be serviced.
  * @return                      True if the request can be serviced.
  */
-static bool PrepareBlockFilterRequest(CNode* pfrom, const CChainParams& chain_params,
+static bool PrepareBlockFilterRequest(CNode& pfrom, const CChainParams& chain_params,
```

Comment:
Does the "p" in `pfrom` stand for "pointer" or "peer"? Consider renaming if the former.

Reply:
I think you're right that the p stands for pointer. I'll rename these if I need to retouch the branch.